src/index.html

```diff
@@ -4,9 +4,327 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"> 
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;1,700&display=swap"
+      rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        src="./images/logo.png"
+        class="nav_logo"
+        alt="logo">
+      <nav class="nav">
+        <ul class="nav__menu">
+          <li class="nav__menu-li">
+            <a class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__menu-li"><a class="nav__link">Samsung</a></li>
```

Comment:
```suggestion
          <li class="nav__menu-li">
            <a href="#" class="nav__link">
               Samsung
             </a>
           </li>
```
Аналогічні виправлення зроби і для інших елементів